Localize/localize-pipeline.yml

```diff
@@ -71,10 +71,10 @@ stages:
 
         git checkout -b $updateBranch
 
-        Remove-Item -Recurse -Force Localize
+        Remove-Item -Recurse -Force Localize, OneLocBuild -ErrorAction Ignore
```

Comment:
Why do we need -ErrorAction Ignore here? What happens if there will be any errors?

Reply:
Well, from the explanation and previous PRs I've gathered that OneLocBuild might not exist, so that way we just ignore if this doesn't exist.
An error will appear pretty much only in case one of the folders doesn't exist(or both) and will delete the one that exists

Reply:
Anyways, fixed to double if for folder checks